### PR TITLE
chore: Improve readability of scopes when using dashed

### PR DESCRIPTION
**Pull request by cweinberger**
Portal2-APi -> Portal2 API
Partner-API -> Partner API
Foo-Bar-Things -> Foo Bar Things

context: https://monta-app01.slack.com/archives/C04HQ96GZ8W/p1699522338224729

### CODE REVIEW

File: src/commonMain/kotlin/com.monta.changelog/log/ChangeLogService.kt [hunk 1]
```diff
@@ -100,7 +100,7 @@ class ChangeLogService(
         return this.commits
             // Group them up by the scope
             .groupBy { commit ->
-                commit.scope.ifHasKebabCaseConvertToCapitalCase()
+                commit.scope.deAndreaImparatonize()
```
Comment (p1mps) [inline src/commonMain/kotlin/com.monta.changelog/log/ChangeLogService.kt:103]:
```suggestion
                commit.scope. ifHasKebabCaseConvertToCapitalCase()
```

maybe :D

File: src/commonMain/kotlin/com.monta.changelog/log/ChangeLogService.kt [hunk 2]
```diff
@@ -126,10 +126,11 @@ class ChangeLogService(
             .toMap()
     }
 
-    private fun String?.ifHasKebabCaseConvertToCapitalCase(): String? {
+    private fun String?.deAndreaImparatonize(): String? {
```
Comment (p1mps) [inline src/commonMain/kotlin/com.monta.changelog/log/ChangeLogService.kt:129]:
```suggestion
    private fun String?. ifHasKebabCaseConvertToCapitalCase(): String? {
```

**Review by p1mps — COMMENTED** [review]
left some suggestions ❤️

File: src/commonMain/kotlin/com.monta.changelog/log/ChangeLogService.kt (hunk 1 above)
Comment (cweinberger) [reply src/commonMain/kotlin/com.monta.changelog/log/ChangeLogService.kt:103]: looks legit :D

**Review by p1mps — APPROVED** [review]

**Pull request merged**